src/containers/deactivateAccount/ducks/types.ts

```diff
@@ -0,0 +1,16 @@
+import { ThunkAction } from 'redux-thunk';
+import { ApiExtraArgs } from '../../../api/apiExtraArgs';
+import { C4CState } from '../../../store';
+import { AsyncRequest } from '../../../utils/asyncRequest';
+import { DeactivateAccountActions } from './actions';
+
+export interface DeactivateAccountReducerState {
+  readonly deactivateAccount: AsyncRequest<void, any>;
+}
+
+export type DeactivateAccountThunkAction<R> = ThunkAction<
+  R,
+  C4CState,
+  ApiExtraArgs,
```

Comment:
I see. We probably additionally should reference this type in `store.ts`

Reply:
I think I copied this from the other ThunkActions, I assumed that it would all just work 😳